src/main/java/org/sagebionetworks/web/client/widget/evaluation/AdministerEvaluationsList.java

```diff
@@ -40,15 +66,24 @@ public AdministerEvaluationsList(AdministerEvaluationsListView view, ChallengeCl
 	 * @param evaluations List of evaluations to display
 	 * @param evaluationCallback call back with the evaluation if it is selected
 	 */
-	public void configure(String entityId) {
+	public void configure(String entityId, Consumer<String> onEditEvaluation) {
 		this.entityId = entityId;
+		this.onEditEvaluation = onEditEvaluation;
 		view.clearRows();
 		synAlert.clear();
+		boolean isInTestWebsite = DisplayUtils.isInTestWebsite(cookieProvider);
+		boolean timeInUtc = globalApplicationState.isShowingUTCTime();
+		String sessionToken = authenticationController.getCurrentUserSessionToken();
+
 		challengeClient.getSharableEvaluations(entityId, new AsyncCallback<List<Evaluation>>() {
 			@Override
 			public void onSuccess(List<Evaluation> evaluations) {
 				for (Evaluation evaluation : evaluations) {
-					view.addRow(evaluation);
+					if(isInTestWebsite && evaluation.getQuota() == null){
```

Comment:
If the evaluation has a quota, your new react component can't be used to render it?  How do we bring your component out of alpha mode?